code/modules/paperwork/pen.dm

```diff
@@ -62,7 +62,7 @@
 		"blue" = list(0.5, 0.5, 1),
 		"yellow" = list(1, 1, 0))
 	var/pen_colour_iconstate = "pencolor"
-	var/pen_colour_shift = 3
+	var/pen_colour_shift = 0
```

Comment:
Autodoc this var and when is this not going to be 0 now?

Reply:
var is now removed as it was unnecessary